### DISCUSSION TITLE
Add "Copy Value" to the field label context menu

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#8542] [Feature Requests]
Clicking the Field Label gives you good options, including a way to open a modal with the raw value... but it would be nice to add a faster _helper_ for copying the value too.

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
